sycl/include/CL/sycl/types.hpp

```diff
@@ -243,11 +243,11 @@ detail::enable_if_t<is_float_to_int<T, R>::value, R> convertImpl(T Value) {
     int OldRoundingDirection = std::fegetround();
     int Err = std::fesetround(FE_TONEAREST);
     if (Err)
-      throw runtime_error("Unable to set rounding mode to FE_TONEAREST");
+      throw std::runtime_error("Unable to set rounding mode to FE_TONEAREST");
```

Comment:
Why have you changed the type to std one?

Reply:
This exception is result of standard function error.

Reply:
Yep, but is thrown by SYCL runtime, not by standard function. I would prefer SYCL exception here

Reply:
OK.
I'm not really sure which `PI_*` error would fit here. Thus, I'll employ PI_ERROR_UNKNOWN.

Reply:
Are you going to change the type?

Reply:
@s-kanaev ping

Reply:
Oh, sorry, I missed the comment.
Yes, the type is changed to `cl::sycl::runtime_error` with `PI_ERROR_UNKNOWN`